modules/mo_pytorch/mo_extensions/front/pytorch/slice_ext.py

```diff
@@ -17,18 +17,17 @@
 from mo.graph.graph import Node
 from mo.ops.strided_slice import StridedSlice
 
-
 class StridedSliceFrontExtractor(FrontExtractorOp):
     op = 'StridedSlice'
     enabled = True
 
     @classmethod
     def extract(cls, node: Node):
-
+        mask = node.module.mask
         attrs = {
-            'begin_mask': [0,1,0,0],
-            'end_mask': [0,1,0,0],
-            'shrink_axis_mask': [0,1,0,0],
+            'begin_mask': mask,
+            'end_mask': mask,
+            'shrink_axis_mask': mask,
```

Comment:
Why the same `mask` is used for multiple different attributes?

Reply:
`begin_mask[i]` equal to 1 means that the corresponding dimension of the begin input is ignored and the real beginning of the tensor is used along corresponding dimension. The same for `end_mask`. This masks will be flipped. 
And if `shrink_axis_mask[i]` is 1, the dimension on the i-th position is deleted, but we should remove the dimension by which we make the slice. 
So we should change `begin_mask`, `end_mask` and `shrink_axis_mask` for the dimension we change.

Reply:
begin_mask[i] equal to 1 means that the corresponding dimension of the begin input is ignored and the real beginning of the tensor is used along corresponding dimension. The same for end_mask. This masks will be flipped.
And if shrink_axis_mask[i] is 1, the dimension on the i-th position is deleted, but we should remove the dimension by which we make the slice.
So we should change begin_mask, end_mask and shrink_axis_mask for the dimension we change.

Reply:
Yes, but what about `a[:, 2:, :, :]`, in example.  This way
```
begin_mask: [0, 1, 0, 0]
end_mask: [0, 0, 0, 0]
```